requirements.txt

```diff
@@ -0,0 +1,5 @@
+dnspython
+matplotlib
+ocflib
+setuptools
+tabulate
```

Comment:
I think you'll also want:
- `mysqlclient`: https://github.com/ocf/utils/blob/f44b7ff9af0e57ea473041eb7d563453c57729ef/makeservices/makemysql-real#L27 Although maybe we can move this to something newer like `pymysql` or a celery task or something in the future.
- `ldap3`: https://github.com/ocf/utils/blob/f44b7ff9af0e57ea473041eb7d563453c57729ef/staff/acct/calnet-breakdowns#L6
- `progressbar`: https://github.com/ocf/utils/blob/f44b7ff9af0e57ea473041eb7d563453c57729ef/staff/acct/calnet-breakdowns#L11
- `pexpect`: https://github.com/ocf/utils/blob/f44b7ff9af0e57ea473041eb7d563453c57729ef/staff/acct/guess-passwords#L14
- `requests`: https://github.com/ocf/utils/blob/f44b7ff9af0e57ea473041eb7d563453c57729ef/staff/acct/guess-passwords#L15
- `lxml`: https://github.com/ocf/utils/blob/f44b7ff9af0e57ea473041eb7d563453c57729ef/staff/sys/apt-dater-ocf#L9
- `paramiko`: https://github.com/ocf/utils/blob/f44b7ff9af0e57ea473041eb7d563453c57729ef/staff/sys/makevm#L21

There's probably also some I'm missing here too.

Reply:
Ooh nice, thanks, that would be quite useful!

Reply:
Hmmm, for some reason it only spat out `ocflib,requests,setuptools`, I think it's because it's only searching for files ending in `*.py` though: https://github.com/asottile/seed-isort-config/blob/55e34634d15a5a04fa92d5b1e963e989f4c96f12/seed_isort_config.py#L71

Reply:
ah yeah indeed, if you have others scripts you'd have to list `--extra path/to/script1 --extra path/to/script2` -- didn't realize this had a bunch of extensionless oneoffs

Reply:
Nice, that worked pretty well (although reasonably messy with all the `extra`s):
```
./venv/bin/seed-isort-config --extra acct/check-dns --extra makeservices/makehttp --extra makeservices/makemysql-real --extra net/pyrc --extra net/weather --extra printing/paper --extra staff/acct/calnet-breakdowns --extra staff/acct/chpass --extra staff/acct/guess-passwords --extra staff/acct/note --extra staff/acct/signat --extra staff/lab/check-toner --extra staff/lab/economode --extra staff/lab/lab-wakeup --extra staff/lab/last-lab --extra staff/lab/ocf-tv --extra staff/lab/timecard --extra staff/mail/extract-mail --extra staff/meetings/bod --extra staff/meetings/minutes --extra staff/sys/apt-dated --extra staff/sys/apt-dater-ocf --extra staff/sys/ldap-lint --extra staff/sys/makevm --extra staff/sys/migrate-vm --extra staff/sys/ocf-marathon --extra staff/sys/ocf-marathon --extra staff/web/bludgeon --application-directories acct:desktop:makeservices:net:printing:staff/acct:staff/lab:staff/mail:staff/meetings:staff/puppet:staff/ssl:staff/sys:staff/web:sys
```

```
[settings]
known_third_party = MySQLdb,dns,ldap3,lxml,matplotlib,ocflib,paramiko,pexpect,progressbar,requests,setuptools,six,tabulate,urllib3
```

Looks like I missed `six` before (I thought it was part of the stdlib), and `urllib3`, although that would be pulled in as a dependency of `requests` anyway.

Reply:
Thanks for digging into this! I think I have covered everything here.